src/reservations/dto/reservation-member.dto.ts

```
@@ -0,0 +1,50 @@
+        description: '사용자 ID',
+        example: 1,
+      })
+      userId: number;
+    
+      @ApiProperty({
+        description: '사용자 닉네임',
+        example: '김파디',
+      })
+      nickname: string;
+    
+      @ApiProperty({
```

ESLint

[error] Delete `····`

(prettier/prettier)